pkg/deploy/service_accounts.go

```diff
@@ -4,21 +4,33 @@
 package deploy
 
 import (
-	"context"
 	"encoding/base64"
 	"fmt"
+	"os"
+	"time"
 
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/go-logr/logr"
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/satoken"
+	authenticationv1 "k8s.io/api/authentication/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
+const (
+	caCertPath      = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	tokenExpiration = time.Hour * 2
+)
```

Comment:
can move these consts inside the function. (since it's only needed by that one func)

Reply:
`tokenExpiration` feels like a magic number, so I like having it as a constant. The cert location is used once, so I'm fine with that being inlined.

Reply:
im not suggesting it to be inlined, just be const inside the function e.g.

```
func (Foo) something() {
   const (
     x = 123
   )
   ...
} 
```